osidb/api_views.py

```diff
@@ -846,6 +846,17 @@ def flaw_available(request: Request, *args, **kwargs) -> Response:
     page, or not?") most likely doesn't change. So this is to prevent public confusion
     during the early stages of security analysis where the preliminary analysis might
     switch between "this CVE affects our products" and "this CVE doesn't affect our products".
+
+    Also an important point is that the client processes CVEs that never get saved to OSIDB's
+    DB (because of internal function `should_create_snippet`), yet the client must
+    publish information about all CVEs. By returning 204 when the flaw doesn't exist in OSIDB's
+    DB, it allows the client to take the output of this API endpoint as actionable advice:
+    When 204, publish the CVE page (either using OSIDB data or using other data), when
+    404, do not publish the CVE page (because the incoming team still works on the CVE).
```

Comment:
Nitpick: I am not sure whether the name incoming team still exists in the RH context.

Reply:
Do we have a better term that is not internal and intuitively understandable?